starter-code/finished_css_code

```diff
@@ -0,0 +1,113 @@
+*{
+  width: 850px;
+}
+
+body{
+  font-family: sans-serif;
+  margin-left: 10%;
+
+}
+
+h1 {
+  color: #FFC963;
+  font-size: 60px;
+  text-decoration-line: none;
+  font-family: sans-serif;
+  font-weight: 200;
+  line-height: 1.6em;
+}
+a {
+  text-decoration: none;
+
+
+}
+
+
+header h1,h2{
+  padding: 0;
+}
+
+h2 {
+  font-size: 30px;
+  font-family: sans-serif;
```

Comment:
You've already set the same font-family in the body so you don't need to set it again here.